source/pug/partials/pulse-projects.pug

```diff
@@ -12,8 +11,9 @@ mixin projects(name, description, url, projectData)
 
 mixin project(meta)
   .col-sm-12.col-md-6.mb-4.mb-md-0
-    img.project-image(src=meta.thumbnail)
-    h4.medium-black.my-2=meta.title
+    a(href=`https://www.mozillapulse.org/entry/${meta.id}`)
```

Comment:
Only because the API is an env variable already. I suppose you could also remove that from being an env var and just point at production, then this would be fine. But leaving the API as an env var opens the door for a disconnect between the link and the entry you're looking at.